### PR TITLE
100% tests coverage for `web5/crypto`

### PR DESCRIPTION
**Pull request by adam4leos**
Screenshot 2023-09-12 at 3 57 46 PM

### CODE REVIEW

File: packages/crypto/src/jose.ts [hunk 1]
```diff
@@ -863,7 +863,7 @@ export class Jose {
     const jose = multicodecToJoseMapping[lookupKey];
 
     if (jose === undefined) {
-      throw new Error(`Unsupported Multicodec to JOSE conversion: '${options.name ?? options.code}'`);
```
Comment (adam4leos) [inline packages/crypto/src/jose.ts:866]: this is unreachable;
we can't hit this line with undefined code AND name, as undefined code would throw in `getNameFromCode` with undefined name

Removed to reach 100% tests coverage, as it's not possible to cover unreachable edge case...

Comment (frankhinek) [reply packages/crypto/src/jose.ts:866]: Appreciate the clarification

**Review by frankhinek — APPROVED** [review]
LGTM
